mubench.reviewsite/src/Controllers/RunsController.php

```diff
@@ -238,7 +238,7 @@ function addRun($experimentId, $detectorId, $projectId, $versionId, $run)
 
         if($this->hasRunTable($detector)){
             $saved_run = Run::of($detector)->in($experiment)->where(['project_muid' => $projectId, 'version_muid' => $versionId])->first();
-            if($saved_run->timestamp !== $run->{'timestamp'}){
+            if(intval($saved_run->timestamp) !== $run->{'timestamp'}){
```

Comment:
The change below says this is an integer column, so why do you need to cast its value to integer?